### DISCUSSION TITLE
TwiML Bins Console Zen migration

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#2378] [Q&A] [answered]
Hi team, 
I've been working on migrating TwiML Bins old react components to Paste components. 
I'd like to get +1 from the design systems team. What's the best way to do this? Here is my figma file for your reference https://www.figma.com/file/kVkKXmg9vYXQAgkAu7SkhE/TwiML-Bins?node-id=0%3A1

**Comment by shleewhite** [comment, marked as answer]
Hi @sang-yeo! 

UX Infrastructure doesn't sign off work at Twilio, because we don't necessarily have all the product context needed to tell you if your designs are correct.  

What we can do is answer questions specific to migration, such as gaps that you've seen in Paste or help you with specific UI controls. If you have any of these kinds of questions, I definitely recommend you sign up for [design system office hours](https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=664218049). We have open time slots tomorrow if you are free!

**Comment by andioneto** [comment]
Hi there @sang-yeo 👋 Do you need more help with regard to this discussion post?

**Comment by sang-yeo** [comment]
No, I'm good thank you for following up :)
